### PR TITLE
Rename environment vars in runtime

### PR DESCRIPTION
**Pull request by ChengTian**
parent https://github.com/aspnet/Coherence-Signed/issues/27

Did successful local `build verify`

@Eilon @muratg @anurse @davidfowl @troydai @NTaylorMullen

### CODE REVIEW

File: scripts/k2.sh [hunk 1]
```diff
@@ -8,12 +8,12 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
...
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-if [ -z "$KRE_APPBASE" ]; then
-    KRE_APPBASE=`pwd`
+if [ -z "$DNX_APPBASE" ]; then
+    DNX_APPBASE=`pwd`
 fi
 
 if [ -f "$DIR/k-$1" ]; then
     exec $DIR/k-$1 "$@"
```
Comment (muratg) [inline scripts/k2.sh]: Not sure what these two lines are, but do they need changing?

**Comment by muratg** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

File: scripts/k2.sh (hunk 1 above)
Comment (ChengTian) [inline scripts/k2.sh]: Talked to @muratg , we'll leave it for now for consistency with k.cmd

**Comment by ChengTian** [comment]
This was merged to dev: https://github.com/aspnet/XRE/commit/8ccf9867fc562fbc9ba8148b9f6fc0aa165bdb62 https://github.com/aspnet/XRE/commit/41c2b4f1e245739a24340ac2fbd11d25351bb1fd

**Pull request closed without merging**